.github/CODEOWNERS

```diff
@@ -30,7 +30,7 @@
 /sdk/batch/                                     @paterasMSFT @dpwatrous
 
 # PRLabel: %Communication
-/sdk/communication/                             @RezaJooyandeh @DominikMe
+/sdk/communication/                             @RezaJooyandeh @DominikMe @Azure/azure-sdk-communication-code-reviewers
```

Comment:
FYI: While this will work on github for PR code reviews using github teams will not help with our pipeline notification errors as we don't have any way to map the team name to an email address to send those notifications to.

Reply:
How does the pipeline notification knows who to send to? I was going to ask you actually :D

Reply:
We can map individual github user accounts to their msft AAD account and that is what we use for sending out pipeline notifications.

Reply:
Nice. Are those individuals selected from the CODEOWNERS file, or is the list stored somewhere else?

Reply:
Those github alias come from the code owners file. See more info at https://azure.github.io/azure-sdk/policies_opensource.html#codeowners.